src/review-list/dialog/review-item.component.tsx

```diff
@@ -35,17 +35,19 @@ const ReviewItem: React.FC<ReviewItemDialogProps> = ({
 }) => {
   const { t } = useTranslation();
 
-  const { encounter, isLoading, isError } = useGetEncounterById(encounterUuid);
+  const { encounter, isLoading } = useGetEncounterById(encounterUuid);
 
   const testsOrder = useMemo(() => {
-    return encounter?.obs.filter((item) => item?.order?.type === "testorder");
-  }, [encounter?.obs]);
+    return encounter?.orders.filter(
+      (item) => item?.orderType.display === "Test Order"
```

Comment:
I have some reservations about this but i hope we'll keep tidying these up with time